elftools/elf/structs.py

```diff
@@ -153,6 +154,16 @@ def _create_shdr(self):
             self.Elf_xword('sh_entsize'),
         )
 
+    def _create_chdr(self):
+        fields = [
```

Comment:
Can you add a comment here that points to the best documentation you know of about this compression?

I don't see this documented in any of the ELF specs I have